chainer/initializations.py

```diff
@@ -0,0 +1,120 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+# Original code forked from MIT licensed keras project
+# https://github.com/fchollet/keras/blob/master/keras/initializations.py
+
+
+def init_weight(weights, initWith, scale=1,
+                none_default=lambda shape: he_normal(shape, 1 / np.sqrt(2))):
```

Comment:
I agree that `none_default` makes the call statement of `init_weight` in some functions simpler. But `none_default` seems redundant if we only look at the interface of `init_weight`. It can be the matter of taste, but I suggest that we remove `none_default` and make the user code be in charge of setting default value of `initWith`.